httpbp/fixtures_test.go

```diff
@@ -116,7 +117,7 @@ func newRequest(t testing.TB) *http.Request {
 	if err != nil {
 		t.Fatal(err)
 	}
-	req.Header.Add(httpbp.EdgeContextHeader, headerWithValidAuth)
+	req.Header.Add(httpbp.EdgeContextHeader, base64.StdEncoding.EncodeToString([]byte(headerWithValidAuth)))
```

Comment:
Should we just have `EncodeEdgeContextHeader` and `DecodeEdgeContextHeader` methods?

Reply:
maybe, but that's more breaking :)

Reply:
I'm fine if you decided to make a future breaking change to clean that up. Right now I just want to fix that bug that's spamming the error log like crazy :)

Reply:
I'm not really sure how that's a breaking change? It's just a way to wrap the `base64` encoding.

Reply:
I assume `EncodeEdgeContextHeader` will replace `NewEdgeContextHeaders` (which does the same thing), and `NewEdgeContextHeaders` is exported so if it goes away that's a breaking change.

Reply:
No, I was just thinking we would have methods to do the b64 encoding/decoding not replace anything 😅